xml/issue2752.xml

```diff
@@ -54,12 +98,12 @@ template &lt;class F, class Allocator&gt;
 <ins>(?) &mdash; A</ins><del>a</del>ny exceptions thrown by the copy or move constructor of <tt>f</tt><ins>.</ins><del>, or </del>
 </li>
 <li>
-<ins>(?) &mdash; For the first version,</ins> <tt>std::bad_alloc</tt> if memory for the internal data structures 
+<ins>(?) &mdash; For the first version,</ins> <tt>std::bad_alloc</tt> if memory for the internal data structures
 could not be allocated<ins>.</ins>
 </li>
 <li>
-<ins>(?) &mdash; For the second version, any exceptions thrown by 
-<tt>std::allocator_traits&lt;Allocator&gt;::template rebind&lt;<em>unspecified</em>&gt;::allocate</tt></ins>.
+<ins>(?) &mdash; For the second version, any exceptions thrown by
+<tt>std::allocator_traits&lt;Allocator&gt;::template rebind_traits&lt;<em>unspecified</em>&gt;::allocate</tt></ins>.
```

Comment:
@AlisdairM Going through rebind_traits instead of rebind should resolve your concern I hope.